Library/Formula/trend.rb

```diff
@@ -0,0 +1,37 @@
+class Trend < Formula
+  desc "a general-purpose, efficient trend graph"
+  homepage "http://www.thregr.org/~wavexx/software/trend/"
+  url "http://www.thregr.org/~wavexx/software/trend/releases/trend-1.2.tar.gz"
+  sha256 "40b0f58101801c9373d2b1d990eb327a4582e9a96e4eb2b43faf87346f9dd64f"
+
+  depends_on :x11
+  depends_on 'freeglut'
```

Comment:
This formula's in the homebrew/x11 tap, not the main repository, so you need to refer to it by its full name: `homebrew/x11/freeglut`

Maybe it makes more sense for this to go to that repo?